apps/api/src/domain/models/photograph/commands/create-photograph/create-photograph.command.integration.spec.ts

```diff
@@ -0,0 +1,219 @@
+import { AggregateType, LanguageCode, MIMEType } from '@coscrad/api-interfaces';
+import { CommandHandlerService } from '@coscrad/commands';
+import { INestApplication } from '@nestjs/common';
+import setUpIntegrationTest from '../../../../../app/controllers/__tests__/setUpIntegrationTest';
+import { CommandFSA } from '../../../../../app/controllers/command/command-fsa/command-fsa.entity';
+import getValidAggregateInstanceForTest from '../../../../../domain/__tests__/utilities/getValidAggregateInstanceForTest';
+import { AggregateId } from '../../../../../domain/types/AggregateId';
+import { DeluxeInMemoryStore } from '../../../../../domain/types/DeluxeInMemoryStore';
+import assertErrorAsExpected from '../../../../../lib/__tests__/assertErrorAsExpected';
+import { clonePlainObjectWithOverrides } from '../../../../../lib/utilities/clonePlainObjectWithOverrides';
+import { ArangoDatabaseProvider } from '../../../../../persistence/database/database.provider';
+import TestRepositoryProvider from '../../../../../persistence/repositories/__tests__/TestRepositoryProvider';
+import generateDatabaseNameForTestSuite from '../../../../../persistence/repositories/__tests__/generateDatabaseNameForTestSuite';
+import { buildTestCommandFsaMap } from '../../../../../test-data/commands';
+import { IIdManager } from '../../../../interfaces/id-manager.interface';
+import { assertCreateCommandError } from '../../../__tests__/command-helpers/assert-create-command-error';
+import { assertCreateCommandSuccess } from '../../../__tests__/command-helpers/assert-create-command-success';
+import { assertEventRecordPersisted } from '../../../__tests__/command-helpers/assert-event-record-persisted';
+import { DummyCommandFsaFactory } from '../../../__tests__/command-helpers/dummy-command-fsa-factory';
+import { CommandAssertionDependencies } from '../../../__tests__/command-helpers/types/CommandAssertionDependencies';
+import buildDummyUuid from '../../../__tests__/utilities/buildDummyUuid';
+import { dummySystemUserId } from '../../../__tests__/utilities/dummySystemUserId';
+import InvalidExternalReferenceByAggregateError from '../../../categories/errors/InvalidExternalReferenceByAggregateError';
+import CommandExecutionError from '../../../shared/common-command-errors/CommandExecutionError';
+import { Photograph } from '../../entities/photograph.entity';
+import { CreatePhotograph } from './create-photograph.command';
+
+const commandType = `CREATE_PHOTOGRAPH`;
+
+const dummyFsa = buildTestCommandFsaMap().get(commandType) as CommandFSA<CreatePhotograph>;
+
+const existingMediaItem = getValidAggregateInstanceForTest(AggregateType.mediaItem).clone({
+    mimeType: MIMEType.png,
+    lengthMilliseconds: undefined,
+});
+
+const titleText = `The Photograph Title`;
+
+const languageCodeForTitle = LanguageCode.English;
+
+const photographer = 'Geoff Horner';
```

Comment:
Hahah, nice.  But maybe we could go back to Jane Doe?

Reply:
Alright. I removed your name.